src/components/Detail/index.scss

```diff
@@ -0,0 +1,248 @@
+@import '../../scss/variables/_colors';
+@import '../../scss/variables/_sizes';
+
+#detail {
+  background-color: $porcelain;
+  height: 100vh;
+  padding: 0 14px;
+  position: absolute;
+  top: 0;
+  transform: translateY(-150%);
+  transition: transform $transition-duration $transition-function;
+  z-index: 1;
+
+  &.visible {
+    transform: translateY(0);
+  }
+}
+
+.button-title {
+  transition: color $transition-duration $transition-function;
+
+  &:hover {
+    color: $blue;
+  }
+}
+
+.big-card {
+  --color: #333;
```

Comment:
This can be moved to sass variables couldn't?

Reply:
It's not possible because i'm modifying this value since JS and with SASS variables it's very complicated to do this